dts/bindings/counter/nxp,mrt.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP Multirate Timer
```

Comment:
Could we use [child-binding](https://docs.zephyrproject.org/latest/build/dts/bindings-syntax.html#child-binding) for the MRT channels? I guess the issue with that might be that the Kconfig symbol depends on both `DT_HAS_xxx_ENABLED` symbols.

Reply:
No because they need to  have devices